src/components/Vote.js

```diff
@@ -40,50 +39,18 @@ const Vote = (props) => {
       let updatedFollowerIds = followers.filter(
         (id) => id !== JSON.stringify(userId)
       )
-      const response = await axios.post(
-        `${apiURL}/graphql`,
-        {
-          query: `
-        mutation {
-          updateUserStory(input: {where: {id: "${story.id}"} data: {followers: [${updatedFollowerIds}]}}){
-            userStory{
-              followers {
-                id
-              }
-            }
-          }
-        }
-        `
-        },
-        {
-          withCredentials: true
-        }
-      )
+      const response = await userStory.updateVotes(story.id, updatedFollowerIds)
       updatedFollowerIds = response.data.data.updateUserStory.userStory.followers.map(
         (follower) => JSON.stringify(follower.id)
       )
       setFollowers(updatedFollowerIds)
       setVoted(false)
       setVotes((votes) => votes - 1)
     } else {
-      const response = await axios.post(
-        `${apiURL}/graphql`,
-        {
-          query: `
-        mutation {
-          updateUserStory(input: {where: {id: "${story.id}"} data: {followers: [${followers}, "${userId}"]}}){
-            userStory{
-              followers {
-                id
-              }
-            }
-          }
-        }
-        `
-        },
-        {
-          withCredentials: true
-        }
+      const response = await userStory.updateStoryVote(
```

Comment:
Both graphql mutations are similar. Lets try to make a single GraphQL mutation by using `updatedFollowerIds` like in the first mutation here.